coordinator/src/main/scala/filodb.coordinator/queryplanner/LogicalPlanParser.scala

```diff
@@ -192,8 +192,13 @@ object LogicalPlanParser {
     val at = sqww.atMs.fold("")(atMs => s" $At${atMs / 1000}")
     val suffix = s"$sqClause$offset$at$ClosingRoundBracket"
     if (sqww.functionArgs.isEmpty) s"$prefix$periodicSeriesQuery$suffix"
-    else {
+    else if (sqww.functionArgs.size <= 1) {
```

Comment:
should this be == 1 since we have already tested for empty above ? ( I know it doesn't matter, but the == check makes it more explicit )